uaclient/util.py

```diff
@@ -100,7 +102,43 @@ def object_hook(o):
         return o
 
 
-def apply_series_overrides(
+def _get_override_weight(
```

Comment:
Unit tests just for this function might be useful and pretty easy to write

Reply:
sure will add